src/burrow.ml

```diff
@@ -535,7 +535,12 @@ let burrow_request_liquidation (p: parameters) (b: burrow) : liquidation_result
       let b_without_reward = { b with collateral = Ligo.sub_tez_tez (Ligo.sub_tez_tez b.collateral partial_reward) creation_deposit } in
       let tez_to_auction = compute_tez_to_auction p b_without_reward in
 
-      if (Ligo.lt_int_int tez_to_auction (Ligo.int_from_literal "0")) || (Ligo.gt_int_int tez_to_auction (tez_to_mutez b_without_reward.collateral)) then
+      (* FIXME: The property checked by the following assertion is quite
```

Comment:
Definitely agree on this point. It could also save us some work in the future if we have to revisit it.

Reply:
Yeah, I think I'll typeset the proof I wrote and add it in comments or somewhere in the spec (in the [operational descriptions](https://checker.readthedocs.io/en/latest/operational.html) maybe?). I'll look into it tomorrow.